unsafe/src/test/java/org/apache/spark/unsafe/map/AbstractBytesToBytesMapSuite.java

```diff
@@ -197,6 +216,74 @@ public void iteratorTest() throws Exception {
   }
 
   @Test
+  public void iteratingOverDataPagesWithWastedSpace() throws Exception {
```

Comment:
Good idea.  I'll fold this into the `emptyMap()` test at the start of this file.